roles/kubernetes/secrets/tasks/gen_certs.yml

```diff
@@ -1,4 +1,24 @@
 ---
+- name: Gen_certs | Create kubernetes config directory (on master[0])
```

Comment:
nit, should we be explicit on the node name?
`master[0]` => `{{ groups[kube-master][0] }}` ?